app/templates/sidebar.html

```diff
@@ -9,7 +9,7 @@ <h1 class="visually-hidden">Admin sidebar</h1>
     <hr>
     <ul class="nav nav-pills flex-column mb-auto">
       <li class="nav-item">
-        <a href="#" class="nav-link text-white {{ 'active' if request.path =='/event_management' }}" aria-current="page">
+        <a href="/events/<term>/" class="nav-link text-white {{ 'active' if request.path =='/events/<term>/' }}" aria-current="page">
```

Comment:
If you need placeholder text, descriptive text would be better, like `/events/current/`

Reply:
changed  placeholder text of the link in `sidebar.html` to `/events/current/`